utils/fasm2frames.py

```diff
@@ -165,7 +170,13 @@ def main():
         db_root_kwargs['required'] = False
         db_root_kwargs['default'] = os.path.join(database_dir, database)
 
+    def_part = os.getenv("XRAY_PART")
```

Comment:
Copy the logic from above about making `--part` required if the env var is not defined.

Reply:
Done.